app/views/helper_questions.html

```diff
@@ -8,11 +8,11 @@
   </li>
 </ul>
 <div id="panel_new" class="bb-tabpanel" role="tabpanel" aria-labelledby="tab1" aria-hidden="false">
-  {% include 'question.html' %}
+  {% include 'question_list_day.html' %}
 </div>
 <div id="panel_active" class="bb-tabpanel" role="tabpanel" aria-labelledby="tab2" aria-hidden="true">
-  <ul data-type="budyup-list">
-    <li>No active question threads.</li>
-  </ul>
+  <section data-type="budyup-list">
+    <p>No active question threads.</p>
```

Comment:
Now that we have this as a paragraph, we should add:

    [data-type="buddyup-list"] p {
        padding: .7rem;
    }